src/components/HelperErrorText/HelperErrorText.tsx

```diff
@@ -38,27 +43,35 @@ export const HelperErrorText = chakra(
         className = "",
         id,
         isInvalid = false,
+        isRenderedText = true,
         text,
         ...rest
       },
       ref?
     ) => {
-      // Only announce the text in the invalid state.
       const styles = useStyleConfig("HelperErrorText", { isInvalid });
       const props = {
         "aria-atomic": ariaAtomic,
         "aria-live": ariaLive === "off" ? undefined : ariaLive,
         className,
         "data-isinvalid": isInvalid,
+        "data-testid": "helperErrorText",
```

Comment:
During testing, Chris had problems with buttons that all had the same `data-testid` value, so I'm concerns that this will cause similar problems.